src/keys.test.js

```diff
@@ -384,6 +385,30 @@ describe("keys", () => {
     });
   });
 
+  describe("fingerprintToFixedLengthHex", () => {
+    it("returns 4-byte zero-padded hex string", () => {
+      const hexOutputs = {
+        '7': '00000007',
+        '123': '0000007b',
+        '2371': '00000943',
+        '53947': '0000d2bb',
+        '322639': '0004ec4f',
+        '16033212': '00f4a5bc',
+        '25201234': '01808a52',
+        '849849884': '32a7ae1c',
+        '8498490984': 'fa8cae68', // should be 1fa8cae68 but truncated front 1
+      };
+
+      for (const xfp of Object.keys(hexOutputs)) {
```

Comment:
Instead of keys, you can key both the key and value using`Object.entries`.
And if you flip the key/values then you could avoid the `Number` coersion.
See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries

Reply:
e.g.:
```      
const hexOutputs = {
        '00000007': 7,
...
```

Reply:
Good call